test/integration/test-video-players-helper.js

```diff
@@ -256,12 +256,16 @@ export function runVideoPlayerIntegrationTests(
   }
 
   function cleanUp() {
```

Comment:
None of this should be necessary. Any iframes appended to the test runner will be cleaned up automatically.